README.md

```diff
@@ -16,14 +16,12 @@ Questions or issues can be [filed as tickets on this repository](https://github.
 This repository contains the front-end code for the Seventh Day Adventist project. [Pattern Lab](http://patternlab.io/) is the tool used for pattern template building and pattern library compilation.
 
 ### Getting Started
-- Use node -v `v7.2.0`
-   - `nvm use 7`
 - [Install Composer globally](https://getcomposer.org/doc/00-intro.md#globally)
 - run `composer install`
-- Set up your local dev url to be `alps.test`
```

Comment:
Why was this removed?

Reply:
This line was moved up, together with composer installation